articles/the-basics-of-genetic-algorithms-in-ml/index.md

```diff
@@ -0,0 +1,75 @@
+A genetic algorithm is a search-based algorithm used for solving optimization problems in machine learning. This algorithm is important because it solves difficult problems that would take a long time to solve. It has been used in various real-life applications such as data centers, electronic circuit design, code-breaking, image processing, and artificial creativity.
+
+This article takes you through the basics of this algorithm and explains how it works. It also explains how it has been applied in various fields and highlights some of its limitations.
+
+### Genetic algorithm (GA) explained 
+The following are some of the basic terminologies that can help us to understand genetic algorithms:
+- **Population:** This is a subset of all the probable solutions that can solve the given problem. 
+- **Chromosomes:** A chromosome is one of the solutions in the population.
+- **Gene:** This is an element in a chromosome.
+- **Allele:** This is the value given to a gene in a specific chromosome. 
+- **Fitness function:** This is a function that uses a specific input to produce an improved output. The solution is used as the input while the output is in the form of solution suitability. 
+- **Genetic operators:** In genetic algorithms, the best individuals mate to reproduce an offspring that is better than the parents. Genetic operators are used for changing the genetic composition of this next generation.  
+
+A genetic algorithm (GA) is a heuristic search algorithm used to solve search and optimization problems. This algorithm is a subset of evolutionary algorithms, which are used in computation. Genetic algorithms employ the concept of genetics and natural selection to provide solution to problems. 
+
+These algorithms have a better intelligence than random search algorithms because they use historical data to take the search to the best performing region within the solution space. 
```

Comment:
```suggestion
These algorithms have better intelligence than random search algorithms because they use historical data to take the search to the best performing region within the solution space.